redis/commands/core.py

```diff
@@ -1069,6 +1069,17 @@ def wait(self, num_replicas, timeout, **kwargs):
         """
         return self.execute_command("WAIT", num_replicas, timeout, **kwargs)
 
+    def hello(self):
+        """
+        Switch to a different protocol, optionally authenticating and setting
```

Comment:
I think there's a miscommunication here. The docs should indicate clearly that we're raising this error, since this is intentionally not being implemented.